docs/01_introduction/README.md

```diff
@@ -143,60 +143,60 @@ Some commonly used optional easyconfig parameters include:
 * ``easyblock``, which specifies which (generic) easyblock should be used;
 * ``sources`` and ``source_urls``, which specify the list of source files and where to download them;
 * ``dependencies`` and ``builddependencies``, which specify (drum roll...) the list of (build) dependencies;
-* ``configopts``, ``buildopts``, and ``installopts``, which speficy options for the configuration/build/install commands, resp.;
+* ``configopts``, ``buildopts``, and ``installopts``, which specify options for the configuration/build/install commands, respectively;
+
+If these parameters are not provided, default values will be provided or extrapolated. For clarity, specifying at least
+which `èasyblock`` to use is a good idea.
 
 ### *Extensions*
 
 *Extensions* is the collective term we use for **additional software packages that can be installed
-on top of another software package**. Examples are *Python packages*, *R libraries* and *Perl modules*
-(can you tell why we had to come up with a different term?).
+on top of another software package**. Examples are *Python packages*, *R libraries* and *Perl modules*.
 
 Extensions can be installed in different ways:
 
 * stand-alone, as a separate installation on top of one or more other installations;
 * as a part of a bundle of extensions that collectively form a separate installation;
-* or as, well, an *extension* to a specific installation to yield a "batteries included"
+* or as an *extension* to a specific installation to yield a "batteries included"
```

Comment:
Maybe using "addition" here is better?

Reply:
I'm not sure I follow?

Reply:
Addition instead of extension? Sounds weird. Hm. Not sure.

Reply:
Let's leave it as is, it's fine.